src/user-statuses/UserStatusScreen.js

```diff
@@ -49,9 +49,9 @@ export default function UserStatusScreen(props: Props): Node {
   const sendToServer = useCallback(
     partialUserStatus => {
       api.updateUserStatus(auth, partialUserStatus);
-      NavigationService.dispatch(navigateBack());
+      navigation.goBack();
```

Comment:
Hmm I guess technically this isn't NFC for the case where there are multiple consecutive `UserStatusScreen`s in the stack. Our `navigateBack` does the thing with `sameRoutesCount`. But we don't expect there to be multiple consecutive `UserStatusScreen`s in the stack, like we do `ChatScreen`s.